examples/domainmodel/src/cli/cli-util.ts

```diff
@@ -0,0 +1,48 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import * as fs from 'fs';
+import colors from 'colors';
+import { AstNode, DefaultDocumentValidator, LangiumDocument, LangiumDocumentConfiguration, LangiumServices } from 'langium';
+
+export function extractDocument(fileName: string, languageId: string, extensions: string[], services: LangiumServices): LangiumDocument {
+    if (!new RegExp(`^.*${extensions?.join('|')}$`).test(fileName)) {
+        console.error(`Please, choose a file with one of these extensions: ${extensions}.`);
+        process.exit(1);
+    }
+
+    if (!fs.existsSync(fileName)) {
+        console.error(`File ${fileName} doesn't exist.`);
+        process.exit(1);
+    }
+    const fileContent = fs.readFileSync(fileName).toString();
```

Comment:
Use `fs.readFileSync(fileName, 'utf-8')` instead of `fs.readFileSync(fileName).toString()`